chatbot-service/src/main/java/com/ages/incuitech/backend/chatbotservice/business/conjunto/desconhecido/regras/IniciarConversaRegra.java

```diff
@@ -31,4 +44,9 @@ public BotMessage processa(MensagemInterna message) {
                 )
         );
     }
+
+    private Map<String, String> buscarNomeDoUsuario(String id) {
+        FacebookProfile perfil = this.facebookService.getProfile(id);
+        return Map.of("nome", perfil.getFirstName(), "sobrenome", perfil.getLastName());
```

Comment:
eu n sabia da existencia desse Map.of...
minha vida javistica foi uma mentira, sempre fazia a criação dos mapas na base do put...... :open_mouth: